pybamm/models/submodels/interface/lithium_plating/plating.py

```diff
@@ -85,26 +105,42 @@ def get_coupled_variables(self, variables):
     def set_rhs(self, variables):
         if self.x_average is True:
             c_plated_Li = variables["X-averaged lithium plating concentration"]
+            c_dead_Li = variables["X-averaged dead lithium concentration"]
             j_stripping = variables[
                 "X-averaged lithium plating interfacial current density"
             ]
-            # Note a is dimensionless (has a constant value of 1 if the surface
-            # area does not change)
             a = variables["X-averaged negative electrode surface area to volume ratio"]
+            L_sei = variables["X-averaged total SEI thickness"]
         else:
             c_plated_Li = variables["Lithium plating concentration"]
+            c_dead_Li = variables["Dead lithium concentration"]
             j_stripping = variables["Lithium plating interfacial current density"]
             a = variables["Negative electrode surface area to volume ratio"]
+            L_sei = variables["Total SEI thickness"]
 
         Gamma_plating = self.param.Gamma_plating
+        # In the partially reversible plating model, coupling term turns reversible
+        # lithium into dead lithium. In other plating models, it is zero.
+        if self.options["lithium plating"] == "partially reversible":
+            dead_lithium_decay_rate = self.param.dead_lithium_decay_rate(L_sei)
+            coupling_term = dead_lithium_decay_rate * c_plated_Li
+        else:
+            zero = pybamm.Scalar(0)
+            coupling_term = zero
```

Comment:
why not just do `coupling_term = pybamm.Scalar(0)` ? This isn't used anywhere else

Reply:
Done